src/main/java/namor/template/calculator/RPNcalculator.java

```diff
@@ -0,0 +1,83 @@
+package namor.template.calculator;
+
+import namor.template.stack.ArrayStack;
+
+public class RPNcalculator {
+    private ArrayStack numbers = new ArrayStack();
+    Sign sign;
+
+    public int calculate(String rpnString){
+        int count = 0;
+        String[] retval = rpnString.split(" ");
+
+         while (retval.length > count){
+            if(retval[count].matches("^[0-9]+$")){
+                numbers.push(Integer.parseInt(retval[count]));
+            } else {
+                switch (sign.getValue(retval[count])){
+                    case ADDITION:
+                        numbers.push(addition());
+                        break;
+                    case SUBTRACTION:
+                        numbers.push(subtraction());
+                        break;
+                    case MULTIPLICATION:
+                        numbers.push(multiplication());
+                        break;
+                    case DIVISION:
+                        numbers.push(division());
+                        break;
+                    default:
+                        throw new IllegalArgumentException("Input elements are wrong!");
```

Comment:
This kind of validation better to hide inside enum. But for now do not use validations at all. You got familiar with exceptions and that's enough for now. I will make task for exception handling in the end.